build broken for android

```
evgeniy@15inch ~/bigdisk1/projects/rust-infra/filetime $ cargo check --target=armv7-linux-androideabi --release
    Updating crates.io index
    Checking filetime v0.2.11 (/home/evgeniy/bigdisk1/projects/rust-infra/filetime)
error[E0425]: cannot find function `futimes` in crate `libc`
    --> src/unix/utimes.rs:35:29
     |
35   |     let rc = unsafe { libc::futimes(f.as_raw_fd(), times.as_ptr()) };
     |                             ^^^^^^^ help: a function with a similar name exists: `futimens`
     | 
    ::: /home/evgeniy/.cargo/registry/src/github.com-1ecc6299db9ec823/libc-0.2.74/src/unix/linux_like/mod.rs:1324:5
     |
1324 |     pub fn futimens(fd: ::c_int, times: *const ::timespec) -> ::c_int;
     |     ------------------------------------------------------------------ similarly named function `futimens` defined here

error[E0425]: cannot find function `lutimes` in crate `libc`
    --> src/unix/utimes.rs:102:19
     |
102  |               libc::lutimes(p.as_ptr(), times.as_ptr())
     |                     ^^^^^^^ help: a function with a similar name exists: `utimes`
     | 
    ::: /home/evgeniy/.cargo/registry/src/github.com-1ecc6299db9ec823/libc-0.2.74/src/unix/mod.rs:1224:5
     |
1224 | /     pub fn utimes(
1225 | |         filename: *const ::c_char,
1226 | |         times: *const ::timeval,
1227 | |     ) -> ::c_int;
     | |_________________- similarly named function `utimes` defined here

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0425`.
error: could not compile `filetime`.

To learn more, run the command again with --verbose.
```

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -46,6 +46,15 @@ jobs:
       run: rustup update stable && rustup default stable && rustup component add rustfmt
     - run: cargo fmt -- --check
 
+  android_build:
+    name: Android build
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Install Rust
+      run: rustup update stable && rustup default stable && rustup target add x86_64-linux-android
+    - run: cargo build --target x86_64-linux-android
+
   publish_docs:
     name: Publish Documentation
     runs-on: ubuntu-latest

--- a/src/unix/mod.rs
+++ b/src/unix/mod.rs
@@ -9,7 +9,6 @@ cfg_if::cfg_if! {
         mod linux;
         pub use self::linux::*;
     } else if #[cfg(target_os = "android")] {
-        mod utimes;
         mod android;
         pub use self::android::*;
     } else if #[cfg(target_os = "macos")] {

